security.tf

```diff
@@ -189,3 +200,22 @@ resource "aws_security_group_rule" "worker_cluster_api_443" {
   to_port                  = 443
   security_group_id        = aws_security_group.workers_security_group[0].id
 }
+
+
+resource "aws_security_group_rule" "vm_private_access_22" {
+
+  count = (length(local.vm_private_access_cidrs) > 0
+    && var.workers_security_group_id == null
+    && ((var.create_jump_public_ip == false && var.create_jump_vm)
+      || (var.create_nfs_public_ip == false && var.storage_type == "standard")
+    )
+    ? 1 : 0
+  )
+  type              = "ingress"
+  description       = "Allow the client VMs to reach a private IP based Jump VM"
```

Comment:
description       = "Allow SSH to a private IP based Jump VM per var.vm_private_access_cidrs.  Req'd for DaC Baseline's client VM."

Reply:
applied